ui/modal/create_password_modal.go

```diff
@@ -136,6 +140,34 @@ func (cm *CreatePasswordModal) SetError(err string) {
 
 }
 
+func handleTabEvent(event chan *key.Event) bool {
+	var isTabPressed bool
+	select {
+	case event := <-event:
+		if event.Name == key.NameTab && event.State == key.Press {
+			isTabPressed = true
+		}
+	default:
+	}
+	return isTabPressed
+}
+
+func (cm *CreatePasswordModal) SwitchEditors(editors ...*widget.Editor) {
+	for i := 0; i < len(editors); i++ {
+		if editors[i].Focused() {
+			cm.isTabPressed = handleTabEvent(cm.keyEvent)
+			if cm.isTabPressed {
+				if i == len(editors)-1 {
+					editors[0].Focus()
+				} else {
+					editors[i+1].Focus()
+				}
+			}
+		}
+		cm.isTabPressed = false
+	}
+}
```

Comment:
This method can be added to the utils.go file, and return a bool value if needed however, the bool value `isTabPressed` seems redundant.

```suggestion
func SwitchEditors(keyEvent *key.Event, editors ...*widget.Editor) {
	for i := 0; i < len(editors); i++ {
		if editors[i].Focused() {
			if handleTabEvent(keyEvent) {
				if i == len(editors)-1 {
					editors[0].Focus()
				} else {
					editors[i+1].Focus()
				}
			}
		}
	}
}
```